client.lua

```diff
@@ -12,13 +8,7 @@ local minimap = RequestScaleformMovie("minimap")
 local hunger
 local thirst
 
-local lastJob = nil
-local PlayerData = nil
-local lastcash = nil
-local lastbank = nil
-local lastdirty = nil
-local lastsociety = nil
-
+local lastJob, lastcash, lastbank, lastdirty, lastsociety = nil, nil, nil, nil, nil
```

Comment:
Does this add much to the optimisation as this is only done once? This may also confuse people as it looks more confusing and I am trying to make this HUD as easy as possible to edit.

Reply:
Shouldn't have to edit those locals and yeah it does some optimisations but not much and also looks neater